core/src/avm2/globals/flash/text/text_field.rs

```diff
@@ -1466,7 +1466,7 @@ pub fn set_scroll_h<'gc>(
             .cloned()
             .unwrap_or(Value::Undefined)
             .coerce_to_i32(activation)?;
-        let clamped = input.clamp(0, this.maxhscroll() as i32);
+        let clamped = input.abs().clamp(0, this.maxhscroll() as i32);
```

Comment:
Could the `clamp(0, this.maxhscroll())` call be changed to `min(this.maxhscroll())` now?

Reply:
Certainly! Nice catch